packages/react-vapor/src/components/tab/TabSelectors.ts

```diff
@@ -0,0 +1,13 @@
+import _ from 'underscore';
+
+import {DEFAULT_GROUP_ID, ITabGroupState, ITabPaneOwnProps} from '.';
+import {IReactVaporState} from '../../ReactVapor';
+
+const getTabGroup = (state: IReactVaporState, ownProps: ITabPaneOwnProps) => {
+    const id = ownProps.groupId ?? DEFAULT_GROUP_ID;
+    return _.find(state.tabs, (currentTabGroup: ITabGroupState) => currentTabGroup.id === id);
```

Comment:
finally the findWhere underscore method is very appropriate here 
`return _.findWhere(state.tabs, {id})` 
or 
```
const getTabGroup = state: IReactVaporState, ownProps: ITabPaneOwnPros) => 
    _.findWhere(state.tabs, {id: ownProps.groupId ?? DEFAULT_GROUP_ID})
```